backend/apps/events/views.py

```
@@ -224,114 +473,66 @@ def get(self, request):
-                return time_error
-
-            # Validar superposición de horarios solo para asistencia
-            overlapping_participating = EventParticipation.objects.filter(
-                user=request.user,
-                event__start_time__lt=end_time,
-                event__end_time__gt=start_time
-            ).exists()
-            
-            if overlapping_participating:
-                return JsonResponse({"detail": "Ya asistes a otro evento en ese horario."}, status=400)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)